PSADHealth/Public/Set-PSADHealthConfig.ps1

```diff
@@ -28,7 +28,7 @@ function Set-PSADHealthConfig
     Param(
 
         [Parameter(Position=0)]
-        $PSADHealthConfigPath = "$(Split-Path $PSScriptRoot)\Config\ADConfig.json",
+        $PSADHealthConfigPath = "$($PSScriptRoot)\Config\ADConfig.json",
```

Comment:
This will work. You can drop the subexpression though. $($PSScriptRoot) can become just $PSScriptRoot